### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by ndviet2307**

### CODE REVIEW

**Comment by ndviet2307** [comment]
ready

**Comment by ndviet2307** [comment]
ready

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/config/credentials.yml.enc
+/config/database.yml
+
+.vscode/settings.json
+
+config/settings.local.yml
+config/settings/*.local.yml
+config/environments/*.local.yml
+/config/application.yml
+/public/uploads/
```
Comment (ghost) [inline .gitignore]: thêm dòng trống cuối file

**Comment by ghost** [comment]
commented
các bạn tạo bản sao cho file database.yml -> database.yml.example  nhé

**Comment by ndviet2307** [comment]
ready

**Comment by ghost** [comment]
commented
banj suwar laij database.yml.example nhuw nay nhes
[code]
default: &default
  adapter: mysql2
  encoding: utf8
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
  username: root
  password:
  socket: /var/run/mysqld/mysqld.sock

development:
  <<: *default
  database: hotel_booking_development

test:
  <<: *default
  database: hotel_booking_test

production:
  <<: *default
  url: <%= ENV["DATABASE_URL"] %>
[/code]

**Comment by ndviet2307** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**